### PR TITLE
[onert] Introduce common CalculateActivationRange

### PR DESCRIPTION
**Pull request by hseok-oh**
This commit introduces CalculateActivationRange in core util.

Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

---

Related issue: https://github.com/Samsung/ONE/issues/8786

### CODE REVIEW

File: runtime/onert/backend/cpu/ops/OperationUtils.h [hunk 1]
```diff
@@ -18,19 +18,20 @@
 #define __NNFW_SUPPORT_NNAPI_OPERATION_UTILS_H__
 
 #include <backend/IPortableTensor.h>
-
```
Comment (hseok-oh) [inline runtime/onert/backend/cpu/ops/OperationUtils.h:21]: This PR includes header include reordering.

**Review by ragmani — DISMISSED** [review]
LGTM

**Review by chunseoklee — APPROVED** [review]

**Comment by hseok-oh** [comment]
I've resolved build fail by using `runtime_error` exception instead of `cout`.

**Review by ragmani — APPROVED** [review]

**Comment by hseok-oh** [comment]
@nnfw-bot test cross-runtime-test-debug

**Review by chunseoklee — APPROVED** [review]

**Pull request merged**